wire-grpc-client/src/main/java/com/squareup/wire/GrpcClient.kt

```diff
@@ -59,23 +62,22 @@ class GrpcClient private constructor(
 
     return Proxy.newProxyInstance(service.java.classLoader, arrayOf<Class<*>>(service.java),
         object : InvocationHandler {
-          override fun invoke(proxy: Any, method: Method, args: Array<out Any>): Any? {
-            val continuation = args.last() as Continuation<Any>
+          override fun invoke(proxy: Any?, method: Method?, args: Array<out Any>?): Any? {
 
             return when (val grpcMethod = methodToService[method] as GrpcMethod<*, *>) {
               is GrpcMethod.RequestResponse -> {
-                continuation.invokeSuspending {
-                  grpcMethod.invoke(continuation, this@GrpcClient, parameter = args[0])
+                (args!!.last() as Continuation<Any>).invokeSuspending {
+                  grpcMethod.invoke(this@GrpcClient, parameter = args[0])
                 }
               }
               is GrpcMethod.StreamingResponse -> {
-                grpcMethod.invoke(continuation, this@GrpcClient, parameter = args[0])
+                grpcMethod.invoke(this@GrpcClient, parameter = args!![0])
```

Comment:
Why does this need the `!!` but the previous call doesn't?

Reply:
The previous call uses bangers when getting the continuation